test/config-files/toml/.markdownlint.toml

```diff
@@ -0,0 +1,2 @@
+[no-trailing-punctuation]
```

Comment:
See my comment elsewhere about not introducing `.markdownlint.toml`, please.

Reply:
Removed the file